crates/common/src/provider.rs

```diff
@@ -254,7 +233,7 @@ impl<'a> From<Cow<'a, str>> for ProviderBuilder {
 pub async fn estimate_eip1559_fees<M: Middleware>(
     provider: &M,
     chain: Option<u64>,
-) -> eyre::Result<(U256, U256)>
+) -> Result<(U256, U256)>
```

Comment:
personally, I prefer the `eyre::` prefix, but because there are no other result types, and this is more or less the default result across the codebase, makes sense